### PR TITLE
Introduce staged build and build argument

### LINKED ISSUE #38: Build container image via multi-staged build via container
## Now

it needs to pre-download required plugin like presto-db2 connector

## Expectation

Use a container image as builder to download required plugin, so that the image build is reproducible.

### PR DESCRIPTION
**Pull request by shawnzhu**
Fixes #38

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,13 +11,24 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
-FROM prestosql/presto:347
 ARG PRESTO_VERSION
 
+FROM debian:buster-slim AS builder
+ARG PRESTO_VERSION
```
Comment (codyjlin) [inline Dockerfile:17]: Is this line and [L14](https://github.com/IBM/docker-prestodb/pull/39?file-filters%5B%5D=.md&file-filters%5B%5D=No+extension&hide-deleted-files=true#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R14) both needed?

File: Dockerfile [hunk 2]
```diff
@@ -11,13 +11,24 @@
...
+
+RUN apt update && \
+    apt -y install wget unzip
+
+RUN wget -c https://github.com/IBM/presto-db2/releases/download/${PRESTO_VERSION}/presto-db2-${PRESTO_VERSION}.zip
```
Comment (codyjlin) [inline Dockerfile:22]: I'm new to [GitHub Releases](https://docs.github.com/en/github/administering-a-repository/about-releases) - didn't know this existed before 😮🆒

**Review by codyjlin — COMMENTED** [review]
Asked a question and will re-review after answer 👍

File: Dockerfile (hunk 1 above)
Comment (shawnzhu) [reply Dockerfile:17]: I'm glad you ask this question. the reason is in this answer: https://stackoverflow.com/a/53682110

File: Dockerfile (hunk 2 above)
Comment (shawnzhu) [reply Dockerfile:22]: I use the github release as artifacts store. need to think about the better option which is github packages which supports maven.

**Review by codyjlin — APPROVED** [review]
LGTM 👍 Thanks for the info

**Pull request merged**